src/main/webapp/app/course/competencies/import-standardized-competencies/course-import-standardized-competencies.component.html

```diff
@@ -1,5 +1,8 @@
 <div class="container">
-    <h2 jhiTranslate="artemisApp.standardizedCompetency.courseImport.title"></h2>
+    <div class="d-flex align-items-center">
+        <h2 jhiTranslate="artemisApp.standardizedCompetency.courseImport.title"></h2>
+        <jhi-documentation-button [type]="documentationType" />
```

Comment:
Shouldn't this link to the Instructor documentation instead? currently, it points to the admins

Reply:
This is intended, as only the admin documentation explains in detail what exactly standardized competenies are, covering the use case where a instructor stumbles across the feature and asks themselves what standardized competencies are.
I will add an info box to the admin documentation that referrs to the instructor documentation.